app/utils.py

```diff
@@ -44,6 +46,8 @@ def format_time(time: Union[int, float]) -> str:
     return f"{time:.2f}{suffix}"  # type: ignore
 
 
+# TODO: better client error & 429 handling
+@retry(stop=stop_after_attempt(7))
```

Comment:
is this triggered from a synchronous flow? or run as a bg job?

Reply:
nvm i see https://github.com/osuAkatsuki/score-service/blob/532073caf49b8d8f22c09c7e17a5531702d4894a/app/api/score_sub.py#L388